### PR TITLE
(node/comcam-mcm.cp) rebuild host as el9

### PR DESCRIPTION
**Pull request by dtapiacl**

### CODE REVIEW

**Review by glennmorris — APPROVED** [review]

File: hieradata/node/comcam-mcm.cp.lsst.org.yaml [hunk 1]
```diff
@@ -1,37 +1,89 @@
...
+  eno2:  #dds interface
+    content:
+      connection:
+        id: "eno2"
+        uuid: "03da7500-2101-c722-2438-d0d006c28c73"
+        type: "ethernet"
+        interface-name: "eno2"
+      ethernet: {}
+      ipv4:
+        address1: "139.229.170.11/24,139.229.170.254"
```
Comment (jhoblitt) [inline hieradata/node/comcam-mcm.cp.lsst.org.yaml:25]: Please add this IP to ipam.

**Review by jhoblitt — CHANGES_REQUESTED** [review]
This is ready to merge but I would like the dds interface added to ipam.

File: hieradata/node/comcam-mcm.cp.lsst.org.yaml (hunk 1 above)
Comment (dtapiacl) [reply hieradata/node/comcam-mcm.cp.lsst.org.yaml:25]: IP was reserved and added to IPAM.

Comment (dtapiacl) [reply hieradata/node/comcam-mcm.cp.lsst.org.yaml:25]: https://ipam.cp.lsst.org/index.php?page=subnets&section=1&subnetId=23

**Review by jhoblitt — APPROVED** [review]
LGTM.

**Pull request merged**